src/Renderer/HtmlRenderer.php

```diff
@@ -256,22 +256,35 @@ public function renderCallStack(Throwable $t, array $trace = []): string
             [],
         );
 
+        $index = 1;
+        if ($t instanceof ErrorException) {
+            $index = 0;
+        }
```

Comment:
Why need different `$index` values?

Reply:
Regular exception:
Index=1
<img width="1433" alt="image" src="https://github.com/user-attachments/assets/e0f4943e-c4df-4423-8234-9c81158cd633">

Index=0
<img width="1471" alt="image" src="https://github.com/user-attachments/assets/64c1a924-d127-4510-89f9-c48665b5d211">

Index=1
<img width="1416" alt="image" src="https://github.com/user-attachments/assets/cb0ca5c5-c149-4822-818c-338ebadd1cbd">

Index=0
<img width="1434" alt="image" src="https://github.com/user-attachments/assets/fde77648-775b-46b6-91a8-2f1b6823dc4e">

Its all because https://github.com/yiisoft/error-handler/blob/20947a8f148ab36df3a62b9fe77037fb31cb0f57/src/ErrorHandler.php#L139 duplicates error message, file and line with stacktrace, but stacktrace has more info such as function arguments. Changing index cause re-render default error list item https://github.com/yiisoft/error-handler/blob/77764e7e1b1037c8c0c8bcab2ebf266ce59b5722/src/Renderer/HtmlRenderer.php#L248